### ISSUE TITLE
Why is there a grey map when creating?

### ISSUE THREAD

**Issue by jmagnusson** [flask-admin/flask-admin#1092]
Right now the geoalchemy leaflet box in admin just a grey box on the create page when there's no default set on the field (see image). I can't seem to zoom out to find some geometry either...

Perhaps we could default all GeoJSONField fields to a certain position if the default position is empty? Somewhere where there's land...

<img width="958" alt="screen shot 2015-10-20 at 11 21 32" src="https://cloud.githubusercontent.com/assets/190835/10603166/c1d00af2-771c-11e5-8324-e13f62c8bd65.png">

My model field looks like:
```python
location = sa.Column(geoalchemy2.Geometry(geometry_type='POINT', srid=4326))
```

I'm using flask-admin 1.3.0.

**Comment by dkerkow** [comment]
The bug is still there in Flask-Admin 1.4.0. 

I think a sensible default would be to center on lat=0, lon=0, zoomlevel 0 and to make it configurable of the scope are of the application is more specific.

**Comment by t4ec** [comment]
Please check #1184

**Comment by pawl** [comment]
Closed by #1184
